src/partials/MusicExercise.js

```diff
@@ -199,6 +201,19 @@ class MusicExercise extends React.Component {
       return <div>Loading</div>
     }
 
+    const engraverParams = {
```

Comment:
Näitä ei voi lisätä .md tiedostosta propseina, joten niiden siirto ulos MusicSheet.js tiedostosta on ongelmallista.

Mahdollinen ratkaisu:
- Default määritelmä MusicSheet.js tiedostoon ja korvataan jos löytyy propseista